src/utils.js

```diff
@@ -128,6 +128,47 @@ export function formatDate(date, variant = 'date') {
 	}
 }
 
+export function getCountdown(date) {
```

Comment:
You can use `formatDate(date, "fromNow")` to display time left in a human-readable format and `differenceInSeconds` from `date-fns` ([here](https://date-fns.org/v2.15.0/docs/differenceInSeconds)) to get the number of seconds left.